### PR TITLE
enable packit

### PR DESCRIPTION
**Pull request by sopos**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#  get-current-version:
+#  - make version
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-all
+    - epel-8
```
Comment (TomasTomecek) [inline .packit.yaml:21]: if you want to have different spec files for jobs, you can define a copr_build job for every target (epel7, epel8, fedora-all) and set `specfile_path` in each:
```yaml
- job: copr_build
  metadata:
    targets: epel-8
  specfile_path: epel8/beakerlib.spec
- job: copr_build
  metadata:
    targets: epel-7
  specfile_path: epel7/beakerlib.spec
- job: copr_build
  metadata:
    targets: fedora
  specfile_path: fedora/beakerlib.spec
```

**Comment by sopos** [comment]
@TomasTomecek , any idea why epel fails? I trier to move files back and forth to dist sub-dir but I'm still getting error about missing patch. Why Fedora patches are ok but epel not?

**Comment by sopos** [comment]
\packit

**Pull request merged**